src/main/java/org/ossgang/commons/observable/Observables.java

```diff
@@ -31,10 +37,77 @@ private Observables() {
      * return the given value on get(). On subscribe(), it will immediately send the given value to the listener.
      *
      * @param value the value
-     * @param <T> the value type
+     * @param <T>   the value type
      * @return a constant ObservableValue
      */
     public static <T> ObservableValue<T> constant(T value) {
         return new ConstantObservableValue<>(value);
     }
+
+    /**
+     * Merges any Observables of the same type into one. Updates whenever all sources have provided at least one item.
+     */
+    @SafeVarargs
+    public static <K, V> ObservableValue<Map<K, V>> zip(Function<K, ObservableValue<V>> supplier, K... items) {
```

Comment:
I think I miss a bit the concepts with the maps (and K keys)? ... What are they for and why would one need the items?

Naively, I would have expected a signature somehow as in Flux:

```
static <T1,T2> Flux<Tuple2<T1,T2>>  zip(ObservableValue<? extends T1> source1, ObservableValue<? extends T2> source2);

```

Reply:
I'll fix that ... it is from where it comes from ;-)